public/app/plugins/datasource/loki/querybuilder/LokiQueryModeller.ts

```diff
@@ -29,25 +29,136 @@ export class LokiQueryModeller extends LokiAndPromQueryModellerBase {
   getQueryPatterns(): LokiQueryPattern[] {
     return [
       {
-        name: 'Log query and label filter',
+        name: 'Log query with parsing',
+        // {} |= `` | logfmt | label="value"
         operations: [
-          { id: LokiOperationId.LineMatchesRegex, params: [''] },
+          { id: LokiOperationId.Logfmt, params: [] },
+          { id: LokiOperationId.LabelFilterNoErrors, params: [] },
+        ],
+      },
+      {
+        name: 'Log query with filtering and parsing',
+        // {} |= `` | logfmt | __error__=``
+        operations: [
+          { id: LokiOperationId.LineContains, params: [''] },
+          { id: LokiOperationId.Logfmt, params: [] },
+          { id: LokiOperationId.LabelFilterNoErrors, params: [] },
+        ],
+      },
+      {
+        name: 'Log query with parsing and label filter',
+        // {} |= `` | logfmt | __error__=''
```

Comment:
Wouldn't 'log query with parsing and label filter' be:

// {} |= `` | logfmt | label="value"

which is what is on line 33? not line 50?

Reply:
Yes, good catch. Updated in [7fbe5db]